backend/src/tests/e2e/signin.test.ts

```diff
@@ -25,8 +25,9 @@ describe('POST /signin', () => {
     });
 
     const req = await request(app).post('/signin').send({
```

Comment:
This case it's not possible. According to our front, we need a test that is for username only and another for email only

Reply:
done